pages/linux/systemctl.md

```diff
@@ -7,21 +7,21 @@
 
 `systemctl --failed`
 
-- Start/Stop/Restart/Reload a service:
+- Start/stop/restart/reload a service:
 
-`systemctl start/stop/restart/reload {{unit}}`
+`systemctl {{start|stop|restart|reload}} {{unit}}`
 
 - Show the status of a unit:
 
 `systemctl status {{unit}}`
 
-- Enable/Disable a unit to be started on bootup:
+- Enable/disable a unit to be started on bootup:
```

Comment:
Don't think this really needs changing? It's fine as-is though.

Reply:
Oh you don't?  It seemed to me incorrect.  If you think so we can revert it.